src/NHSD.BuyingCatalogue.Solutions.Application/Commands/NativeDesktop/UpdateSolutionNativeDesktopOperatingSystems/UpdateSolutionNativeDesktopOperatingSystemsCommand.cs

```diff
@@ -0,0 +1,19 @@
+using MediatR;
+using NHSD.BuyingCatalogue.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Application.Commands.Validation;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.Commands.NativeDesktop.UpdateSolutionNativeDesktopOperatingSystems
+{
+    public sealed class UpdateSolutionNativeDesktopOperatingSystemsCommand : IRequest<ISimpleResult>
+    {
+        public string SolutionId { get; }
+
+        public string NativeDesktopOperatingSystemsDescription { get; }
+
+        public UpdateSolutionNativeDesktopOperatingSystemsCommand(string solutionId, string nativeDesktopOperatingSystemsDescription)
+        {
+            SolutionId = solutionId.ThrowIfNull();
+            NativeDesktopOperatingSystemsDescription = nativeDesktopOperatingSystemsDescription?.Trim(); ;
```

Comment:
there is an extra semicolon at the end of this line

Reply:
✔️